test/e2e/workload/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+- janetkuo
+- nikhiljindal
+- kargakis
+- mfojtik
+reviewers:
+- janetkuo
```

Comment:
@kargakis Should delete the other people, and only "sig-apps-reviewers" is enough ?